recipe/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "torcheval" %}
+{% set version = "0.0.7" %}
+{% set commit_id = "f1bc22fc67ec2c77ee519aa4af8079f4fdaa41bb" %}
```

Comment:
This doesn't look like the right commit to me. The right one is 1aaa7ff0e4fa3aae9e1a66a1b56780fa5d1651c9. Traced from https://github.com/pytorch/torcheval/actions/runs/5969005517.

Reply:
fair enough, good catch!

https://github.com/pytorch/torcheval/compare/f1bc22fc67ec2c77ee519aa4af8079f4fdaa41bb...1aaa7ff0e4fa3aae9e1a66a1b56780fa5d1651c9

considering that the GH sources are used only for tests and requirements.

Reply:
Yeah, nothing major. Just a reminder to be careful when there is no tag on the upstream repo.